### PR TITLE
Add explain button next to run query

### PR DESCRIPTION
**Pull request by prajwal-pai77**
Adds a explain button next to Run query

### CODE REVIEW

**Review by AayushTyagi1 — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -958,6 +973,10 @@
           "command": "vscode-couchbase.showFavoriteQueries",
           "when": "false"
         },
+        {
+          "command": "vscode-couchbase.explainQuery",
+          "when": "false"
+        },
```
Comment (lokesh-couchbase) [inline package.json:979]: Keep it with run query

File: src/extension.ts [hunk 2]
```diff
@@ -903,6 +904,26 @@ context.subscriptions.push(disposable);
...
+        queryHistoryTreeProvider,
+        true
+      );
+      vscode.commands.executeCommand(
+        "setContext",
+        "vscode-couchbase.runButtonEnabled",
+        true
+      );
+    })
+  );
```
Comment (lokesh-couchbase) [inline src/extension.ts:925]: Make sure, explain button is also disabled if query is running

Comment (lokesh-couchbase) [reply src/extension.ts:925]: I haven't tested this though

Comment (prajwal-pai77) [reply src/extension.ts]: I have tested it, when we hover when running a query, explain query button cannot be clicked & vice versa

File: package.json (hunk 1 above)
Comment (prajwal-pai77) [reply package.json]: Done

**Review by lokesh-couchbase — APPROVED** [review]

**Pull request merged**